content/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust.md

```diff
@@ -38,6 +39,37 @@ We recommend that you have a basic understanding of the Rust language. For more
 1. On the "Rust - by {% data variables.product.prodname_actions %}" workflow, click **Configure**.
 
    ![Screenshot of the "Choose a workflow" page. The "Configure" button on the "Rust" workflow is highlighted with an orange outline.](/assets/images/help/actions/starter-workflow-rust.png)
+
+{%- ifversion ghes %}
```

Comment:
This section is for GitHub Enterprise Server users who may not have access to GitHub Marketplace.